src/is-registered.ts

```diff
@@ -1,8 +1,8 @@
 
 export const isRegistered = (name: string) => {
-  switch (document.createElement(name).constructor) {
-    case HTMLElement: return false;
-    case HTMLUnknownElement: return false;
+  switch (document.createElement(name).constructor.name) {
```

Comment:
Since IE doesn't support function names, this fix won't work.
But AFAIK the underlying problem isn't the component registration, it's the modification of the function/class name when minifying/uglifying the js/ts. For that, we could just apply #20 and this should not be needed.